src/plugin/lib/rules/conditions.js

```diff
@@ -0,0 +1,178 @@
+const MESSAGE_COUNTER_CLOCKWISE_COMPARISON = 'You should use only clockwise comparison `<`';
```

Comment:
Please, add specs for changes:
https://eslint.org/docs/developer-guide/nodejs-api#ruletester